components/forward_post_modal/forward_post_modal.tsx

```diff
@@ -0,0 +1,303 @@
+        if (bodyRef.current) {
+            setBodyHeight(bodyRef.current.getBoundingClientRect().height);
+        }
+    };
+
+    const selectedChannelId = selectedChannel?.details?.id || '';
+
+    const canPostInSelectedChannel = useSelector(
+        (state: GlobalState) =>
+            Boolean(selectedChannelId) &&
+            haveIChannelPermission(
+                state,
+                selectedChannel?.details?.team_id || '',
+                selectedChannelId,
+                Permissions.CREATE_POST,
+            ),
+    );
+    const isPrivateConversation = channel.type !== General.OPEN_CHANNEL;
+    const canForwardPost = (isPrivateConversation || canPostInSelectedChannel) && !postError;
+
+    const onHide = useCallback(() => {
+        // focusPostTextbox();
+        onExited?.();
+    }, [onExited]);
+
+    const handleChannelSelect = useCallback(
+        (channel: ValueType<ChannelOption>) => {
+            if (Array.isArray(channel)) {
+                setSelectedChannel(channel[0]);
+            }
+            setSelectedChannel(channel as ChannelOption);
+        },
+        [],
+    );
+
+    // since the original post has a click handler specified we should prevent any action here
+    const preventActionOnPreview = (e: React.MouseEvent) => {
+        e.preventDefault();
+        e.stopPropagation();
+    };
+
+    const messagePreviewTitle = formatMessage({
+        id: 'forward_post_modal.preview.title',
+        defaultMessage: 'Message preview',
+    });
+
+    const previewMetaData: PostPreviewMetadata = {
+        post,
+        post_id: post.id,
+        team_name: currentTeam.name,
+        channel_display_name: channel.display_name,
+        channel_type: channel.type,
+        channel_id: channel.id,
+    };
+
+    let notification;
+    if (isPrivateConversation) {
+        let notificationText;
+        if (channel.type === General.PRIVATE_CHANNEL) {
+            const channelName = `~${channel.display_name}`;
+            notificationText = (
+                <FormattedMessage
+                    id='forward_post_modal.notification.private_channel'
+                    defaultMessage='This message is from a private channel and can only be shared with <strong>{channel}</strong>'
+                    values={{
+                        channelName,
+                        strong: (x: React.ReactNode) => <strong>{x}</strong>,
+                    }}
+                />
+            );
+        } else {
+            const allParticipants = channel.display_name.split(', ');
+            const participants = allParticipants.map((participant) => <strong key={participant}>{participant}</strong>);
+
+            notificationText = (
+                <FormattedMessage
+                    id='forward_post_modal.notification.dm_or_gm'
+                    defaultMessage='This message is from a private conversation and can only be shared with {participants}'
```

Comment:
line 3259 has the <strong> tag surrounding {participants} and this instance doesn't.